Problems/prob013/models/ProgressivePartyProblem.essence

```diff
@@ -8,13 +8,13 @@ letting Boat be domain int(1..n_boats)
 
 given capacity, crew : function (total) Boat --> int(1..)
 
-find hosts : set of Boat,
+find hosts : set (minSize 1) of Boat,
      sched : set (size n_periods) of function (total) Boat --> Boat
 
 minimising |hosts|
 
 such that
-    forAll p in sched . range(p) subsetEq hosts,
+    forAll p in sched . range(p) = hosts,
```

Comment:
The line below says that every value in `hosts` is represented in `p`, and the line above has just said that `p` is a subset of `hosts`, so they must be equal.